client/landing/jetpack-cloud/sections/backups/index.js

```diff
@@ -29,6 +31,7 @@ import {
 
 export default function () {
 	if ( config.isEnabled( 'jetpack-cloud/backups' ) ) {
+		FullStory.init( { orgId: TRACKING_IDS.jetpackCloudFullStory } );
```

Comment:
To go with the above, this needs to become:

```suggestion
		initFullStory( { orgId: TRACKING_IDS.jetpackCloudFullStory } );
```